src/pages/ConciergePage.tsx

```diff
@@ -56,7 +56,7 @@ function ConciergePage({session}: ConciergePageProps) {
 
     return (
         <ScreenWrapper testID={ConciergePage.displayName}>
-            <View style={[styles.borderBottom]}>
+            <View style={[styles.borderBottom, styles.appContentHeader]}>
```

Comment:
@nkdengineer After the inclusion of `styles.appContentHeader`, the height of the 2nd skeleton Header is smaller than the first skeleton Header and actual Header.

https://github.com/user-attachments/assets/ef892ca7-e5da-4c0b-bdbc-d2d461bc9300


https://github.com/user-attachments/assets/b90e9a3a-bd10-42e3-aa8f-6fd3c4b4baba

Reply:
@sobitneupane I updated to fix this case.